template-parts/excerpt/excerpt-video.php

```diff
@@ -9,16 +9,13 @@
  * @since 1.0.0
  */
 
-// Print the 1st video we can find.
-if (
-	! twenty_twenty_one_print_first_instance_of_block( 'core/embed', get_the_content() ) &&
-	! twenty_twenty_one_print_first_instance_of_block( 'core/video', get_the_content() )
-) {
+$content = get_the_content();
 
-	// Fallback to the content if no block was found.
-	the_content();
```

Comment:
If there is no `core/video` or `core/embed`/`core-embed/*` block we'll need to fallback to `the_content()` instead of `the_excerpt()`

Reply:
@melchoyce & I just talked this through, and since these templates are only used when the "excerpt" view is shown, it doesn't make sense to show the full content in some cases. We should show the audio/video/embed if we can find it, and the excerpt always.

Based on that convo I've also updated the other `excerpt-*` templates, since only aside, status, and link should use `the_content`.

Reply:
This is a problem....
There are hundreds of sites that have older content, and the current approach will only work for block-based posts.
If the site has posts that are not block-based, then `the_excerpt` will not render anything.
You can test this for example by importing the test XML file from https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml which contains old post-format posts.
Falling-back to `the_content` instead of `the_excerpt` will take into account those sites.

I added a PR against your fork with the tweaks I had in mind (https://github.com/ryelle/twentytwentyone/pull/2)
I apologize for the extra files in the diff, I pulled from trunk first

Reply:
> There are hundreds of sites that have older content, and the current approach will only work for block-based posts. That's why we had the the_content() fallbacks.

I'm okay with this. This theme should show off the best of new features and show off blocks. Older posts aren't broken — you can convert the older content to blocks if you want your site to take advantage of the block-formatted post format excerpts on archive pages. And if that's a problem, the option to show all of your content exists via the Customizer setting.

Reply:
This PR changes the syntax and makes an awesome job at making it more readable for developers. But it doesn't actually "fix" anything for users, in fact it breaks backwards-compatibility. 
Is improving readability for developers worth a clear regression for end-users/visitors?

Reply:
This theme is designed to show off the block editor rather than the classic editor. If folks want to use the classic editor, there are previous default themes available that look great. If they want their site to support post formats with this theme, there are a few plugins I can recommend to convert old classic content to blocks.